dao/seeds/source_types.yml

```
@@ -37,6 +37,12 @@ amazon:
         - type: required
         - type: pattern
           pattern: "^[A-Za-z0-9]+[A-Za-z0-9_-]*$"
+      - component: text-field
+        name: application.extra.bucket_region
+        label: S3 bucket region
+        validate:
+          - type: pattern
+            pattern: "^[a-z]+[a-z0-9-]*$"
```

```suggestion
        - type: pattern
          pattern: "^[a-z]+[a-z0-9-]*$"
```

These just need to be moved in by one indentation (array of objects)

---

No problem. YAML allows leading white space for lists but I'll change it to match the rest of the file.